.ci/jobs/apm-agent-nodejs-mbp.yml

```diff
@@ -12,8 +12,8 @@
         discover-pr-origin: merge-current
         discover-tags: true
         notification-context: 'apm-ci'
-        # Run CI builds only on main, PRs, and non-EOL version branches.
-        head-filter-regex: '^(main|PR-.*|3\.x)$'
+        # Run CI builds only on main, PRs, non-EOL version branches, and tags.
+        head-filter-regex: '^(main|PR-.*|3\.x|v3.3[0-9].*|v[4-9].*)$'
```

Comment:
<img width="433" alt="image" src="https://user-images.githubusercontent.com/2871786/157840237-0c5214ae-5b25-4194-9865-516893ff5020.png">

Reply:
If we get to v3.40.0 then it will fail this regex, right?